filenames.gypi

```diff
@@ -135,6 +135,7 @@
       'atom/browser/api/atom_api_protocol.h',
       'atom/browser/api/atom_api_render_process_preferences.cc',
       'atom/browser/api/atom_api_render_process_preferences.h',
+      'atom/browser/api/atom_api_screen_mac.mm',
```

Comment:
This line should be added after `'atom/browser/api/atom_api_screen.h',` to preserve the correct ordering.